eng/pipelines/pr.yml

```
@@ -116,16 +116,72 @@ stages:
           agentPool: VSEngSS-MicroBuild2022-1ES
         testType: Functional
         testTargetFramework: net472
-        timeoutInMinutes: 60
+        testProjectName: Msbuild.Integration.Test
 
 - ${{ if eq(parameters.RunFunctionalTestsOnWindows, true) }}:
   - stage:
-    displayName: Functional Tests on Windows (.NET 8.0)
+    displayName: Functional Tests on Windows (.NET Framework 4.7.2) NuGet.CommandLine.FuncTest
     dependsOn: []
     jobs:
     - template: pr.job.yml
       parameters:
-        displayName: Functional Tests on Windows (.NET 8.0)
+        displayName: Functional Tests on Windows (.NET Framework 4.7.2) NuGet.CommandLine.FuncTest
+        osName: Windows
+        ${{ if eq(variables['System.TeamProject'], 'public') }}:
+          agentPool: NetCore-Public
+          agentDemands: ImageOverride -equals Windows.VS2022Preview.Amd64.Open
+        ${{ else }}:
+          agentPool: VSEngSS-MicroBuild2022-1ES
+        testType: Functional
+        testTargetFramework: net472
+        testProjectName: NuGet.CommandLine.FuncTest
+
+- ${{ if eq(parameters.RunFunctionalTestsOnWindows, true) }}:
+  - stage:
+    displayName: Functional Tests on Windows (.NET Framework 4.7.2) NuGet.CommandLine.Test
+    dependsOn: []
+    jobs:
+    - template: pr.job.yml
+      parameters:
+        displayName: Functional Tests on Windows (.NET Framework 4.7.2) NuGet.CommandLine.Test
```

The stage names are kind of lengthy already. Can the job name be shortened so it's easier to see the job title in the AzDo interface?

```suggestion
        displayName: NuGet.CommandLine.Test
```

---

How about `NuGet.CommandLine.Test on Windows (.NET Framework 4.7.2)` ?  Some tests run on multiple platforms and frameworks so I think those details need to be part of the name because that's how test results are labeled.

---

Updated!